packages/toolchain/collection.yaml

```diff
@@ -21,22 +21,32 @@ packages:
       autobump.revdeps: "true"
   - name: "luet-makeiso"
     category: "toolchain"
-    version: 0.3.8-15
+    version: 0.3.8-16
     upx: false
     fips: false
     labels:
       github.repo: "luet-makeiso"
       github.owner: "mudler"
       autobump.revdeps: "true"
+    requires:
+    - name: "luet"
+      category: "toolchain"
+      # Would be nice to restrict to the luet version defined within this file
+      version: ">=0"
   - name: "luet-makeiso"
     category: "toolchain-fips"
-    version: 0.3.8-16
+    version: 0.3.8-17
     upx: false
     fips: true
     labels:
       github.repo: "luet-makeiso"
       github.owner: "mudler"
       autobump.revdeps: "true"
+    requires:
+    - name: "luet"
```

Comment:
```suggestion
    - <<: *luet
```

Reply:
I saw that in `cos` package, but honestly I did not understand it and I did not want to mess with something I do not understand. Where is this logic taking place? Is this part of luet? Part of the yaml parser? Shouldn't this line `<<: *luet` replace the whole object (I mean `name`, `category`, `version`...)?

Reply:
Oh cool, I'll give it a try